packages/menu/src/MenuItem.tsx

```diff
@@ -88,7 +88,7 @@ const titleTextStyle = css`
   }
 `;
 
-const activetitleTextStyle = css`
+const activeTitleTextStyle = css`
```

Comment:
Brooke fixed both of the typos in this file in her PR. Let's omit these changes so we keep the concerns separated.

Reply:
This was a complete oversight (ditto the mention of it in the changeset file). Thanks for noticing this.